### DISCUSSION TITLE
Is there a way to pass custom styles to the prev/next/end buttons?

### DISCUSSION THREAD

**Discussion by iuricernov** [hakimio/ngx-ui-tour#73] [General]
Hi,
I want to have custom colors for the prev/next/end buttons, but I didn't find yet a simple solution.
Do someone worked on something like this before?

**Comment by hakimio** [comment]
No, there are no plans to add this functionality. You have to use [custom template](https://hakimio.github.io/ngx-ui-tour/md-menu/Misc#custom-template). It is a pretty simple solution.

**Reply by iuricernov** [reply]
Oh great, this is what I need. Thanks!

**Reply by edu22snt** [reply]
Were you able to implement it?

If you did, can you show me step by step how you did it?

I need to implement something custom but I can't.

**Reply by iuricernov** [reply]
Trying to refresh my mind here:
1- Create a tour step component, ex: `yourapp-tour-step-template`. On the TS file you add TourService as dependency, and copy the custom template provided to the html.
2- Instead of using `<tour-step-template>`, use `<yourapp-tour-step-template>`
3- Check if everything is still working, and change the tour step component accordingly.

**Reply by edu22snt** [reply]
Do you have an example? If you have, please show me?

**Reply by edu22snt** [reply]
I'm having a lot of difficulty implementing it.
